pdf-tools-install fail

-*- mode: compilation; default-directory: "~/.emacs.d/elpa/27.0/develop/pdf-tools-20190309.744/build/server/" -*-
Comint started at Mon Mar 11 13:06:58

/Users/chenxianbin/.emacs.d/elpa/27.0/develop/pdf-tools-20190309.744/build/server/autobuild -i /Users/chenxianbin/.emacs.d/elpa/27.0/develop/pdf-tools-20190309.744/
---------------------------
    Installing packages    
---------------------------
brew install pkg-config poppler automake
Warning: pkg-config 0.29.2 is already installed and up-to-date
To reinstall 0.29.2, run `brew reinstall pkg-config`
Warning: poppler 0.74.0 is already installed and up-to-date
To reinstall 0.74.0, run `brew reinstall poppler`
Warning: automake 1.16.1_1 is already installed and up-to-date
To reinstall 1.16.1_1, run `brew reinstall automake`

---------------------------
 Configuring and compiling 
---------------------------
./configure -q --bindir=/Users/chenxianbin/.emacs.d/elpa/27.0/develop/pdf-tools-20190309.744/ && make -s
configure: error: Package requirements (poppler-glib >= 0.16.0) were not met:

Package 'libffi', required by 'gobject-2.0', not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables poppler_glib_CFLAGS
and poppler_glib_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
===========================
     Build failed.  ;o(    
===========================
Note: maybe try the '-d' option.

Comint exited abnormally with code 1 at Mon Mar 11 13:07:01

I have a similar error on Mac:
```
./configure -q --bindir=/Users/sergiobacelar/.emacs.d/elpa/pdf-tools-20190413.2018/ && make -s
configure: error: Package requirements (poppler-glib >= 0.16.0) were not met:

Package 'libffi', required by 'gobject-2.0', not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables poppler_glib_CFLAGS
and poppler_glib_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
===========================
     Build failed.  ;o(    
===========================
Note: maybe try the '-d' option.
```

--- a/server/autobuild
+++ b/server/autobuild
@@ -369,6 +369,9 @@ os_macos() {
         PKGARGS=install
         PACKAGES="pkg-config poppler automake"
         PKG_INSTALL_AS_ROOT=
+        # homebrew install libffi as keg-only, meaning we need to set
+        # PKG_CONFIG_PATH manually so configure can find it
+        export PKG_CONFIG_PATH="$(brew --prefix libffi)/lib/pkgconfig/"
     elif which port >/dev/null 2>&1; then
         PKGCMD=port
         PKGARGS=install

